src/locales/zh/common.json

```diff
@@ -48,6 +48,7 @@
     "category": "分类",
     "optional": "可选",
     "0disable": "0 到禁用"
+    "millisecond": "毫秒 | 毫秒"
```

Comment:
@xkonglong @Kholid060  comma missing in both #1679 and 1678

<img width="495" alt="image" src="https://github.com/AutomaApp/automa/assets/15135943/c4932b03-a7f0-47d3-b6bc-2b5d0e01e08d">

<img width="516" alt="image" src="https://github.com/AutomaApp/automa/assets/15135943/0209d8b9-b9e1-4b55-bd5a-9d46b132f4c0">